src/Sylius/Behat/Context/Ui/Admin/ManagingPromotionsContext.php

```diff
@@ -229,4 +283,15 @@ private function assertFieldValidationMessage($element, $expectedMessage)
             sprintf('Promotion %s should be required.', $element)
         );
     }
+
+    /**
+     * @param PromotionInterface $promotion
+     * @param string $field
+     */
+    private function assertIfFieldIsTrue(PromotionInterface $promotion, $field)
+    {
+        $this->updatePage->open(['id' => $promotion->getId()]);
+
+        Assert::true($this->updatePage->hasResourceValues([$field => 1]), sprintf('Promotion %s is not exclusive, but it should be.', $promotion->getName()));
```

Comment:
You have error message ``'Promotion %s is not exclusive, but it should be.'``, but this method is used also in ``thePromotionShouldBeCouponBased``.